src/onemod/backend/jobmon_backend.py

```diff
@@ -231,7 +231,7 @@ def evaluate_with_jobmon(
 
     # Set config
```

Comment:
I think the config arg for jobmon should be a path, not a config object (hence the previous `model.dataif.config`)

Reply:
They should both be the path to the pipeline config file. Where does it expect a `PipelineConfig` object?

Reply:
Just below, at https://github.com/ihmeuw-msca/OneMod/blob/release/1.0/src/onemod/backend/jobmon_backend.py#L236.

Because if `model` is a `Pipeline`, then `model.config` would be a `PipelineConfig` - https://github.com/ihmeuw-msca/OneMod/blob/release/1.0/src/onemod/pipeline.py#L42. This attribute on `Pipeline` has been the case since the initial redesign commits I think.

Reply:
- `config` is an argument for `evaluate`, and should be the path (string) to the pipeline config JSON file: https://github.com/ihmeuw-msca/OneMod/blob/ec384608a9bbe8815112c76749b717111fbec470/src/onemod/main.py#L124
- Here's the command line argument that Jobmon uses to call `evaluate`: https://github.com/ihmeuw-msca/OneMod/blob/ec384608a9bbe8815112c76749b717111fbec470/src/onemod/backend/jobmon_backend.py#L143
- Here's what `task_args` used to look like, where `config` is a string: https://github.com/ihmeuw-msca/OneMod/blob/1a91608e2faede8adc43c5c5de746e9f4907c834/src/onemod/backend/jobmon_backend.py#L223

I think it might be confusing that I used "config" to mean both the JSON file and the config objects for `Pipeline` and `Stage`; maybe we should use `config_path` or something different to make it more obvious?